src/Octoshift/GithubApi.cs

```diff
@@ -375,12 +375,12 @@ public virtual async Task<string> StartOrganizationMigration(string sourceOrgUrl
             return (string)data["data"]["startOrganizationMigration"]["orgMigration"]["id"];
         }
 
-        public virtual async Task<(string State, string SourceOrgUrl, string TargetOrgName, string FailureReason)> GetOrganizationMigration(string migrationId)
+        public virtual async Task<(string State, string SourceOrgUrl, string TargetOrgName, string FailureReason, int? RemainingRepositoriesCount, int? TotalRepositoriesCount)> GetOrganizationMigration(string migrationId)
```

Comment:
In what scenario the remaining repo count and total repo could would be null? can we just rely on zero as an exceptional case and work with that?

Reply:
We discussed this on Slack, but just adding an answer here to for the record.

At the moment, the API returns `0` when the count has not yet been computed. I have filed a bug, because I think we should change it to return `null` in this case. This code is designed to support both cases.